signal_analog/flow.py

```diff
@@ -142,6 +146,85 @@ def find_publish(statement):
             raise ProgramDoesNotPublishTimeseriesError(statements)
 
 
+class Plot(object):
+    """Represents a Plot for a Chart as configured in the SignalFx UI.  It is a helper class that makes it simpler
+       to use lower-level abstractions like Assign, Function, Data, publish()"""
+
+    def __init__(self, assigned_name, signal_name, filter=None, rollup=None, fx=None, label=None, visible=True,
+                 extrapolation_policy=None, max_extrapolations=None):
+        """Represents a Plot for a Chart as configured in the SignalFx UI.  It is a helper class that makes it simpler
+        to use lower-level abstractions like Assign, Function, Data, publish()
+
+        Other options available in the UI for a plot include: display_units, plot_color, and visualization_type.  These
+        values can be configured in with_publish_label_options() in the Chart itself.  See PublishLabelOptions.
+
+        Example:
+
+        >>> TimeSeriesChart() \
+        >>>    .with_name("Cpu Utilization") \
+        >>>    .with_program(
+        >>>       Plot("A", "CPUUtilization", filter, rollup="max", fx=[Mean(by="app")])
+        >>> )
+
+        Arguments:
+            assigned_name: the assigned name used in the SignalFlow program, use A-Z to keep compatibility with the
```

Comment:
> use A-Z to keep compatibility

It _has_ to be `A` to `Z` right? I don't think you can label it anything outside of that range. Might want to include a check here to prevent someone from naming it something else.

Reply:
Actually, it doesn't have to be A-Z.  But it is a good idea for charts.  

For detectors, it is actually nicer if you don't use A-Z because you get nicer UX when you say things like `errors_in_last_hour`.

Reply:
Ah gotcha, thanks for clarifying! No need to change anything here then.